### ISSUE TITLE
Possible improvement in documentation of cross-compile

### ISSUE THREAD

**Issue by chitrak7** [snowleopard/hadrian#513]
In the documentation for building cross-compiling GHC, the steps mentioned are : 
git clone https://github.com/ghc/ghc
cd ghc
git clone https://github.com/snowleopard/hadrian

this step will result in an error at multiple sub-repositories with the error similar this :
fatal: repository 'https://github.com/ghc/packages/deepseq.git/' not found.

It is because of GitHub's  policy to not include '/' in names. This can be easily removed by the following command.
git config --global url."https://github.com/ghc/packages-".insteadOf https://github.com/ghc/packages/

I think this should be included in the documentation.

@snowleopard 
Also, if this change is approved, can I put the PR myself?

**Comment by snowleopard** [comment]
@chitrak7 Yes, sure, go ahead! Documentation improvements are very welcome. Please coordinate with @angerman and @izgzhen who are taking care of cross compilation. I myself will not be able to properly review the suggested changes.

Also note that @angerman is working on a huge PR that improves cross compilation: #445. We hope to merge this PR soon, so some things may be changing soon.

**Comment by izgzhen** [comment]
> this step will result in an error at multiple sub-repositories with the error similar this :
fatal: repository 'https://github.com/ghc/packages/deepseq.git/' not found.

BTW, I tried the newest source today, and met some problems finding submodule like Cabal. I fixed it by cloning from haskell.org instead of github.

**Comment by angerman** [comment]
I would strongly suggest removing any mention of cloning GHC from github.

**Comment by izgzhen** [comment]
> I would strongly suggest removing any mention of cloning GHC from github.

Agreed
